desc/plotting.py

```diff
@@ -2153,11 +2464,22 @@ def plot_field_lines_sfl(eq, rho, seed_thetas=0, phi_end=2 * np.pi, ax=None, **k
     seed_thetas : float or array-like of floats
         Poloidal positions at which to seed magnetic field lines.
         If array-like, will plot multiple field lines.
+    phi_start: float
+        Toroidal angle to integrate field line from, in radians. Default is 0.
     phi_end: float
         Toroidal angle to integrate field line until, in radians. Default is 2*pi.
+    phi_end: float
```

Comment:
Should this be `phi_spacing` or something similar? And this input should be used to set the `dphi` variable, which is currently undefined.

Reply:
thanks, this should be `dphi` and `dphi=1e-2` is the default value (from the function signature)